GLMakie/Project.toml

```diff
@@ -40,3 +40,6 @@ Printf = "1.0, 1.6"
 ShaderAbstractions = "0.5"
 StaticArrays = "0.12, 1.0"
 julia = "1"
+
+[sources]
+Makie = { path = ".." }
```

Comment:
Should ```
[sources]
ReferenceTests = {path = "../../ReferenceTests"}
``` be added to `test/Project.toml` (same remark for `CairoMakie` and `WGLMakie`) ?

I think `ReferenceTests = "d37af2e0-5618-4e00-9939-d430db56ee94"` can be removed from `GLMakie/Project.toml` (only used in tests).